demo/3000-home/next.config.js

```diff
@@ -3,6 +3,8 @@ const NextFederationPlugin = require('@module-federation/nextjs-mf');
 module.exports = {
   webpack(config, options) {
     if (!options.isServer) {
+      config.module.rules.push(...NextFederationPlugin.loaders);
```

Comment:
We should push these into webpack via the apply step of the pluing

compiler.options.rules.push()

Reply:
I didn't work for some reason. But I tried again, and it works! Thanks. Fix https://github.com/module-federation/nextjs-mf/pull/103/commits/f1d19b6c71c56ba03cf8963ffa8dc6fba3817822

<img width="949" alt="Screen Shot 2022-09-15 at 11 09 42" src="https://user-images.githubusercontent.com/1946920/190320085-ecc4c3d5-1913-4857-8c0d-4cec081a2010.png">

But loader removing led to the addition of a new extraOption `disableMFClient`. Users should be able to completely disable the next router patching with MFClient. Before it can be done via removing loader in next.config.js how I wrote here https://github.com/module-federation/nextjs-mf/pull/103#issuecomment-1247264089

Reply:
agreed, should be controlable via options - id change the name to something else though (mentioned in other comment)